lib/livebook/runtime/definitions.ex

```diff
@@ -65,6 +65,11 @@ defmodule Livebook.Runtime.Definitions do
     dependency: %{dep: {:stb_image, "~> 0.6.2"}, config: []}
   }
 
+  xlsx_file = %{
+    name: "xlsx_file",
+    dependency: %{dep: {:xlsx_reader, "~> 0.8.3"}, config: []}
```

Comment:
```suggestion
  xlsx_reader = %{
    name: "xlsx_reader",
    dependency: %{dep: {:xlsx_reader, "~> 0.8.3"}, config: []}
```

Reply:
Will refactor to package-name instead of file-type